x/gov/module.go

```diff
@@ -150,8 +156,8 @@ func (am AppModule) LegacyQuerierHandler(legacyQuerierCdc *codec.LegacyAmino) sd
 
 // RegisterServices registers module services.
 func (am AppModule) RegisterServices(cfg module.Configurator) {
-	v1beta1.RegisterMsgServer(cfg.MsgServer(), keeper.NewMsgServerImpl(am.keeper))
-	v1beta1.RegisterQueryServer(cfg.QueryServer(), am.keeper)
+	v1beta2.RegisterMsgServer(cfg.MsgServer(), keeper.NewMsgServerImpl(am.keeper))
+	v1beta2.RegisterQueryServer(cfg.QueryServer(), am.keeper)
```

Comment:
Should we add v1beta1 too, for backwards-compatibility?

Reply:
Yes sorry, we want both.